src/routes/gas/getEnrichedGases.ts

```diff
@@ -0,0 +1,36 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { enrichedGas } from '../../types/gas.types';
+import { Type } from '@sinclair/typebox';
+import { getEnrichedGases } from '../../lib/gas';
+
+const schema = {
+  desription: 'Get enriched gases',
+  tags: ['gas'],
```

Comment:
```suggestion
  tags: ['gas price'],
```

Käytetään samaa kuin luonti-endpointissakin :D